src/main/java/solitaire/graphics/ImageLoader.java

```diff
@@ -13,12 +13,14 @@
     public static BufferedImage backTexture; //This is the texture of the back of a card
     public static BufferedImage emptySpotTexture; //This is the texture of a stack when it's empty
 
+    public static int cardTexWidth;
+    public static int cardTexHeight;
 
 
     public static void init(String resFolderPath){
         cardTextures = new BufferedImage[52];
-        int cardTexWidth = 40; //2.5 * 16
-        int cardTexHeight = 56; //3.5 * 16
+        cardTexWidth = 100; //2.5 * 16
```

Comment:
We should make these variables `public static final int`s and move the assignment to the declaration.